assets/js/comments.js

```diff
@@ -1,5 +1,13 @@
 var video_data = JSON.parse(document.getElementById('video_data').textContent);
 
+var isRTL = (() => {           
```

Comment:
This looks a bit messy tbh. Maybe just look at the html language and see if matches one of the rtl languages?
(Like here: https://github.com/iv-org/invidious/blob/164d764d553921e6ee98facda241f13c2103ec90/src/invidious/helpers/i18n.cr#L191-L198)